### ISSUE TITLE
Contrast questions

### ISSUE THREAD

**Issue by awkawk** [w3c/wcag#675]
From the list: https://lists.w3.org/Archives/Public/public-comments-wcag20/2019Mar/0003.html

Your guidelines provide a contrast threshold above which the contrast is considered acceptable. Here are my questions:

1. Is maximum contrast (black & white) the best for the broadest cross section of viewers?
2. Is there any reason other than aesthetics to use lower levels of contrast?
3. Why is gray text used at all?
4. If maximum contrast can accommodate the widest audience, including older people, why not use it?
5. Many people claim that high contrast is bad for reading or the eyes, the so-called "eye strain". Is that true and are there any studies to support that? Or is it possibly a subjective judgement.

FYI:Wired Magazine article about gray text
https://www.wired.com/2016/10/how-the-web-became-unreadable/

The pages at your site all seem to use maximum or very high contrast.
Please let me know your thoughts.

**Comment by patrickhlauke** [comment]
2, 3, 4: it's not the place of WCAG to mandate design/aesthetics. and it's not the place of WAI to comment on the motivation of designers (why they would choose particular color combinations). color (and shades/tints of color, such as gray text) is a useful tool (together with size, layout, spacing, etc) to establish visual hierarchy, provide differentiation, guide a user's attention, etc. there are valid reasons for designers to choose something other than just pure black on pure white.

1, 5: yes, certain users can find contrast that is too high problematic. WCAG does not have any SCs that deal with this, currently.

**Comment by Myndex** [comment]
Hi @awkawk Hi @patrickhlauke 

Some of these issues are being discussed and researched in issue 695. but for some fast answers:

**Regarding 1, 4, 5** Maximum contrast can NOT accommodate the widest audience. Older eyes have increasing problems with scatter and glare, and excessive high contrast can make this worse. Also excessive contrast can increase viewer fatigue. 

It is a fallacy that simply increasing contrast improves legibility across the board — human visual perception and the vast array of impairments are much more complicated than such a simple solution.

The #1 way to improve legibility in my opinion (research pending) is to increase font size such that the "circle of confusion" of light on the retina is smaller than the width of a font's lowercase stem or bar. 

FAA Human Factors states that 15:1 is a reasonable maximum contrast. I just posted a more in depth discussion of font size vs contrast in 695.

**Regarding 2:** There are a ton of reasons to use various contrasts, and some of them are functional not simply aesthetic. But then you also get into "where is the line" between functional and aesthetic?

This goes so deep into principles of design, I'm not even going to try to answer, but I will offer some references for further study:

Itten's classic "The Elements of Color." There is some about Itten I disagree with, but he is foundational and essential reading.
https://monoskop.org/images/4/46/Itten_Johannes_The_Elements_of_Color.pdf

On Typography, Bringhurst's well regarded "The Elements of Typography"
https://alejandrocarpintero.files.wordpress.com/2016/08/02_the_elements_of_typographic_style.pdf

The above applied to the Web:
http://webtypography.net/intro/

On Art/design in general: If you want to be a designer, that means you want to be an artist[1]. FIRST STEP is take **life drawling classes that teach the Nicolaides method**. I even tell photographers that life drawing is the most important class. You may have no interest in drawing, but still LIFE DRAWING IS THE MOST IMPORTANT CLASS.

Why? Life Drawing isn't so much about learning how to draw, **it's about learning you how to SEE**, by that I mean, how to look at things as an artist. All artists — actor, photographer, painter, web designer, Solidworks drafter — all will benefit from taking — and retaking over the years — life drawing classes.

https://www.goodreads.com/book/show/3061.The_Natural_Way_to_Draw

_[1] I define "artist" as anyone who creates "something" with the intent to engage "someone" and especially if the intent is to **communicate a viscerally.**_

**Comment by DavidMacDonald** [comment]
I think we have the right balance now. 

However, I'd like to see some work on our algorithm so that we don't have that weird situation were a few failing combinations appear worse than some of the passing combinations to the human eye.  I'm looking at you Jared :)

**Comment by Myndex** [comment]
> I think we have the right balance now.
> However, I'd like to see some work on our algorithm so that we don't have that weird situation were a few failing combinations appear better than some of the passing combinations to the human eye.

Hi @DavidMacDonald thank you for commenting. 

To be honest the current algorithm is **far** from balanced, and not accurate at all, actually. It is not supported by much of the existing research or world-wide standards on contrast for accessibility or visual perception.

It is is being discussed more in depth in issue #695, and there are ample examples there demonstrating the deficiencies, as well as a path toward a more workable solution. Comments welcome on the work I am doing on this issue over there.

**Comment by alastc** [comment]
Proposed response, mostly summarising the comments above:
<hr>

Issue #665 was raised due to a discussion with the author of the wired article, which is aiming to bring font-weight & typeface into the contrast metrics in future.


For the questions:

> 1. Is maximum contrast (black & white) the best for the broadest cross section of viewers?

No, certain users can find contrast that is too high problematic. There is [some research](https://www.w3.org/WAI/RD/2012/text-customization/r11) that non-white backgrounds can help certain users. WCAG does not have any guidelines that deal with this, currently.

> 2. Is there any reason other than aesthetics to use lower levels of contrast?
> 3. Why is gray text used at all?

It depends where you draw the line between aesthetics, branding and other factors. However, the guidelines are aimed at preventing issues that affect people with disabilities. There is always room for improvement, but they should not interfere with design choices except within that remit.

> 4. If maximum contrast can accommodate the widest audience, including older people, why not use it?
> 5. Many people claim that high contrast is bad for reading or the eyes, the so-called "eye strain". Is that true and are there any studies to support that? Or is it possibly a subjective judgement.

There are enough people who use fairly extreme measures to avoid maximum contrast to believe that it is a significant issue for some. (E.g. tinted glasses, screens over their monitor, or software that overrides the output.) Some for what you might categories as cognitive issues, some due to visual impairments. 

However, whether there should be a guideline to prevent maximum contrast is a slightly different question. There is a certain amount of user control that is easier to apply to too-much contrast, compared to not-enough. If someone made a solid case for a guideline about too much contrast it would be considered.

**Comment by Myndex** [comment]
Hi @alastc 
> However, whether there should be a guideline to prevent maximum contrast is a slightly different question. There is a certain amount of user control that is easier to apply to too-much contrast, compared to not-enough. If someone made a solid case for a guideline about too much contrast it would be considered.

This is certainly part of the process in the experiments for #695 — There are existing guidelines and standard though as well. FAA Human Factors specifies the following:

3:1 — MINIMUM contrast.
7:1 — PREFERRED or recommended contrast
15:1 — MAXIMUM contrast.

On the subject of maximum contrast, as I have different impairments in each eye I can discuss my own issues as they relate to this.

My left eye, with a Symfony IOL implant has a substantial vitreous detachment and floaters., some of which tend to "hang out" right in front of the fovea. The right eye with a CrystalLens IOL has developed a membrane (soon to be removed by YAG) so at the moment that eye has both lower contrast and increased scatter similar to the cataract the IOL replaced.

YUK!!

But it is helping me evaluate these conditions of contrast (I am trying to get through a lot of this before the membrane is removed in a couple weeks, LOL).

In short:

1. Excessive contrast causes SCATTER, glare, and increased chromatic aberration. While counter intuitive, too much contrast will actually cause a decrease of acuity especially in older eyes where scatter is an increasing concern.

**Comment by Myndex** [comment]
Hi @alastc 
> However, whether there should be a guideline to prevent maximum contrast is a slightly different question. There is a certain amount of user control that is easier to apply to too-much contrast, compared to not-enough. If someone made a solid case for a guideline about too much contrast it would be considered.

This is certainly part of the process in the experiments for #695 — There are existing guidelines and standard though as well. FAA Human Factors specifies the following:

3:1 — MINIMUM contrast.
7:1 — PREFERRED or recommended contrast
15:1 — MAXIMUM contrast.

On the subject of maximum contrast, as I have different impairments in each eye I can discuss my own issues as they relate to this. I am taking this over to 695 however as it relates more to those studies.

**Comment by alastc** [comment]
The proposed response above was agreed by the working group.

**Comment by Myndex** [comment]
Just to add in closing: "too much" contrast is less of an issue in practical instances than "too little", nevertheless, this issue is subsumed as part of the 695 research and is anticipated to be part of the  recommendations resulting from that research.

**Comment by alastc** [comment]
Hi @Myndex, do you have a reference to hand for the impact of too much contrast? I've googled around FAA & contrast, but I'm not finding anything directly applicable.

**Comment by Myndex** [comment]
> Hi @Myndex, do you have a reference to hand for the impact of too much contrast? I've googled around FAA & contrast, but I'm not finding anything directly applicable.

HI @alastair, yes it is in the FAA human factors standards, screenshots below.

The 15:1 they mention here is I believe roughly equivalent to the _WCAG math_ of 9:1 to 10:1 when the brightest color is #FFF.

![Screen Shot 2019-06-24 at 6 22 25 PM](https://user-images.githubusercontent.com/42009457/60066582-7441d400-96bc-11e9-9e38-b87b8aa7a96f.png)

And also my personal experiments - very high contrasts causes noticeable halation, glare, scatter, and chromatic aberration, and sometimes discomfort. Other research indicates this increases with age (I'm 55).

Nevertheless, I consider "max contrast" as an issue at **near the bottom of the issues** I am examining. Total luminance is more important to fatigue and discomfort for instance. If you set a monitor to 300 nits, and have an #FFF bg and #000 text in a dark room it is quite uncomfortable to look at. 

Also, chromatic aberration is more of a problem with good acuity/good sharpness. Poor acuity reduced chromatic aberration due to the blur. Glare though, and scatter, are real problems especially for those over 40.

### The same document provides this guidance on contrast which is similar to ISO's:

![Screen Shot 2019-06-24 at 6 47 59 PM](https://user-images.githubusercontent.com/42009457/60067122-36de4600-96be-11e9-85c4-490f074c1e17.png)

### On the subject of color pairings:

![Screen Shot 2019-06-24 at 7 11 02 PM](https://user-images.githubusercontent.com/42009457/60067199-86247680-96be-11e9-97c3-d0fa3e8bbcb3.png)




### References/where to find:
![Screen Shot 2019-06-24 at 6 22 42 PM](https://user-images.githubusercontent.com/42009457/60066993-c46d6600-96bd-11e9-9695-a70706cfbab8.png)

![Screen Shot 2019-06-24 at 6 23 17 PM](https://user-images.githubusercontent.com/42009457/60067004-ca634700-96bd-11e9-92a5-f28bf1f2c2c0.png)
